packages/documentation/src/stories/components/accordion/accordion-item.stories.ts

```diff
@@ -4,12 +4,17 @@ const meta: Meta = {
   id: '3b86bc9b-3dcd-4788-a436-88fd18a6312d',
   title: 'Components/Accordion Item',
   component: 'post-accordion-item',
+  parameters: {
+    controls: {
+      exclude: [
+        'heading-level', // the heading level should be set on the parent post-accordion
```

Comment:
Instead of removing it visually from the docs, I would suggest to leave it there, but add a deprecation message.
What do you think?